logs/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore everything in this directory
+ 
+# Except this file…
+!.gitignore
+
```

Comment:
I would put the ignore to the top level file.
```
/logs/
```
Matches the top level directory and ignores everything inside it. But then Makefile would need a target to create the directory. I don't know which is the better solution.

Reply:
Nor do I! I considered both.

Reply:
One reason to use top level file would be if dockerignore was auto generated from gitignore. But docker build uses only libdds subdirectory making this solution equal for that purpose.